motr/io_req_fop.c

```diff
@@ -163,28 +165,32 @@ static void application_attribute_copy(struct m0_indexvec *rep_ivec,
 	m0_ivec_cursor_init(&ti_cob_cursor, ti_ivec);
 	m0_ivec_cursor_init(&ti_goff_cursor, ti_goff_ivec);
 
-	rep_index	= m0_ivec_cursor_index(&rep_cursor);
-	ti_cob_index	= m0_ivec_cursor_index(&ti_cob_cursor);
+	rep_index = m0_ivec_cursor_index(&rep_cursor);
+	ti_cob_index = m0_ivec_cursor_index(&ti_cob_cursor);
+	ti_goff_index = m0_ivec_cursor_index(&ti_goff_cursor);
 
 	/* Move rep_cursor on unit boundary */
 	off = rep_index % unit_size;
 	if (off) {
-		if (m0_ivec_cursor_move(&rep_cursor, unit_size - off)) {
+		if (!m0_ivec_cursor_move(&rep_cursor, unit_size - off))
 			rep_index = m0_ivec_cursor_index(&rep_cursor);
+		else
+			return;
+	}
+	off = ti_cob_index % unit_size;
+	if (off) {
```

Comment:
coding style issue.

```
137   * use difference between NULL, 0 and "false" to emphasize whether
138     an expression is used as a pointer, integer (including success
139     or failure code) or boolean:
140
141           if (p == NULL) { /* assumes that p is a pointer */
142           } else if (q != 0) { /* q is an integer */
143           } else if (r) { /* r is a boolean */
144           }
145

```

Please write it as:
if (off != 0) {

Reply:
Fixed